meldebot/mel/conf.py

```diff
@@ -103,30 +104,30 @@ def init_logger(verbose, debug):
     logging.basicConfig(level=log_level, format=log_format)
 
 
-def get_telegram_token():
-    return config.defaults().get("telegram_token", False)
+def get_telegram_token() -> Optional[str]:
+    return config.defaults().get("telegram_token")
 
 
-def get_giphy_api_key():
-    return config.defaults().get("giphy_api_key", False)
+def get_giphy_api_key() -> Optional[str]:
+    return config.defaults().get("giphy_api_key", None)
 
 
-def get_tenor_api_key():
-    return config.defaults().get("tenor_api_key", False)
+def get_tenor_api_key() -> Optional[str]:
+    return config.defaults().get("tenor_api_key", None)
 
 
-def get_image_server_auth():
+def get_image_server_auth() -> Dict[str, Any]:
     return {
         'user': config.defaults().get("image_server_user", False),
         'password': config.defaults().get("image_server_password", False)
     }
 
 
-def get_store_path():
+def get_store_path() -> Any:
```

Comment:
This one should replicate the same as the above ones
```suggestion
def get_store_path() -> Optional[str]:
```